### DISCUSSION TITLE
2.0 RC is merged to main!

### DISCUSSION THREAD

**Discussion by cseickel** [nvim-neo-tree/neo-tree.nvim#183] [Announcements]
I just merged all of the 2.x commits to main. I would really appreciate feedback from everyone currently following main, or anyone that opts into the v2.x branch (which is the same thing right now), even if it is just a 👍 or 👎 on this announcement. Thank you in advance!

The breaking changes are actually quite small, but there are several deprecations. There are enough changes to the config that you are likely to see a warning message about these changes, **but nothing should break**. What should happen is that your old config should be automatically remapped onto the new one and it will just give you a warning about it. I'd really like to know if you like the way this was done or if you have any issues with it.

Please see https://github.com/nvim-neo-tree/neo-tree.nvim/wiki/Changelog for details on what has changed.

Once I am satisfied that there are no issues, 2.x will become the new default.

**Comment by hbiel** [comment]
Hey!

I just updated my config to the v2.x branch. Thanks for your work on this!

A minor thing that I noticed is that some of the global mappings are not suitable for example the git_status source. This error message is shown:

`[neo-tree.nvim] [ERROR 20:48:56] .../packer/start/neo-tree.nvim/lua/neo-tree/ui/renderer.lua:557: Could not find command close_all_nodes for mapping z in git_status`

I'll let you when I find anything else!

**Reply by cseickel** [reply]
Thanks for pointing that out! That command should definitely be universal.

**Reply by cseickel** [reply]
That's fixed now.
